mcast.md

```diff
@@ -0,0 +1,96 @@
+## Multicast application demo
+The purpose of the document is to illustrate the experiment done to validate
+multicast in a contiv network.
+
+### Steps to run sender and receiver multicast application between containers
+For this experiement we will run 2 containers , one on each VM. We will create a vlan network
+and run a sender and receiver multicast application. For the purpose of this demo we will be using
+multicast application from https://github.com/leslie-wang/py-multicast-example
+
+#### Step 1: Pull the contiv netplugin workspace from github
+```
+$ git clone https://github.com/contiv/netplugin
+$ cd netplugin
+```
+
+#### Step 2: Run the demo script
+`$ make demo`
+
+#### Step 3: Login to netplugin-node1
+`$ vagrant ssh netplugin-node1`
+
+#### Step 4: Create a network
+`$ sudo netctl net create contiv-net --encap=vlan --subnet=20.1.1.0/24 --gateway=20.1.1.254 --pkt-tag=1010`
+
+#### Step 5: Pull a docker multicast application box from dockerhub
```

Comment:
too many steps, perhaps it is better to consolidate them in a few steps each step with multiple operations.. for e.g. 4 steps with substeps within can be:
- start vms
- create multicast enabled network
- run multicast receiver on node1
- run multicast sender on node2